src/packages/@ncigdc/modern_components/ClinicalAnalysis/ClinicalAnalysisResult.js

```diff
@@ -284,80 +290,69 @@ const ClinicalAnalysisResult = ({
-                  margin: '0 1rem 1rem',
-                  padding: '0.5rem 1rem 1rem',
+                  margin: '5px 0 10px',
                 }}
                 >
-                <div
+                <h2
                   style={{
-                    margin: '5px 0 10px',
+                    fontSize: '1.8rem',
+                    marginBottom: 0,
+                    marginTop: 10,
                   }}
                   >
-                  <h2
-                    style={{
-                      fontSize: '1.8rem',
-                      marginBottom: 0,
-                      marginTop: 10,
-                    }}
-                    >
                     Overall Survival
-                  </h2>
-                </div>
-                <div
-                  style={{
-                    height: '250px',
-                    margin: '5px 2px 10px',
-                  }}
-                  >
-                  <SurvivalPlotWrapper
-                    {...overallSurvivalData}
-                    height={430}
-                    plotType="clinicalOverall"
-                    slug={OVERALL_SURVIVAL_SLUG}
-                    survivalDataLoading={survivalDataLoading}
-                    uniqueClass="clinical-survival-plot"
-                    />
-                </div>
-              </Column>
+                </h2>
+              </div>
+              <div
+                style={{
+                  height: '250px',
+                  margin: '5px 2px 10px',
+                }}
+                >
+                <SurvivalPlotWrapper
+                  {...overallSurvivalData}
+                  height={430}
+                  plotType="clinicalOverall"
+                  slug={OVERALL_SURVIVAL_SLUG}
+                  survivalDataLoading={survivalDataLoading}
+                  uniqueClass="clinical-survival-plot"
+                  />
+              </div>
+            </li>
 
-              {setId && map(displayVariables, (varProperties, varFieldName) => {
-                const filters = {
-                  content: [
-                    {
-                      content: {
-                        field: 'cases.case_id',
-                        value: [`set_id:${setId}`],
-                      },
-                      op: '=',
+            {setId && map(displayVariables, (varProperties, varFieldName) => {
+              const filters = {
+                content: [
+                  {
+                    content: {
+                      field: 'cases.case_id',
+                      value: [`set_id:${setId}`],
                     },
-                  ],
-                  op: 'and',
-                };
+                    op: '=',
+                  },
+                ],
+                op: 'and',
+              };
 
-                return varProperties.plotTypes === 'continuous'
-                  ? (
+              return (
+                <li
+                  className="masonry-brick"
```

Comment:
this will re-introduce mounting issues with the cards. list items need to have keys

```suggestion
                  className="masonry-brick"
                  key={varFieldName}
```